### PR TITLE
Improves project documentation and other nitpicks

### PR DESCRIPTION
**Pull request by gruyaume**
Improves project documentation and other nitpicks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,105 +2,40 @@
...
 
 ## Usage
 
-The Hydra Operator may be deployed using the Juju command line as follows:
-
-Deploy the `postgresql-k8s` charm:
-
 ```bash
 juju deploy postgresql-k8s --channel edge --trust
+juju deploy hydra --trust
```
Comment (natalian98) [inline README.md:11]: there's no need to deploy with `--trust`

**Review by nsklikas — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (gruyaume) [reply README.md:11]: We're patching the k8s service aren't we? This requires k8s credentials, hence the `--trust`

Comment (natalian98) [reply README.md:11]: True, it's also [suggested](https://github.com/canonical/hydra-operator/blob/main/lib/charms/observability_libs/v0/kubernetes_service_patch.py#L235) in the service patch lib, but the service gets patched even without trusting the charm. I asked about it in [observability](https://chat.charmhub.io/charmhub/pl/jbdijf8z97rqpcihjkimfxbmzw).

Comment (gruyaume) [reply README.md:11]: Yeah it depends on your k8s cluster ultimately. If you deploy your charm on GKE or other cloud k8s, you will end up having those errors:
```
unit-grafana-k8s-0: 08:29:58 ERROR unit.grafana-k8s/0.juju-log Unable to patch the Kubernetes service: Failed to patch k8s service: (403)
Reason: Forbidden
```

File: README.md [hunk 2]
```diff
@@ -2,105 +2,40 @@
...
-
-## Contributing
-
-Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this
-charm following best practice guidelines, and
-[CONTRIBUTING.md](https://github.com/canonical/hydra-operator/blob/main/CONTRIBUTING.md) for developer guidance.
-
-## License
-
-The Charmed Hydra Operator is free software, distributed under the Apache Software License, version 2.0. See [LICENSE](https://github.com/canonical/hydra-operator/blob/main/LICENSE) for more information.
```
Comment (natalian98) [inline README.md:106]: Why should these points be removed? Adding part of it was suggested in this [pr](https://github.com/canonical/hydra-operator/pull/13#issuecomment-1335392767)

File: README.md (hunk 1 above)
Comment (natalian98) [reply README.md:11]: That's a good point, I'll add `trust` to integration tests and kratos as well. Thanks

File: README.md (hunk 2 above)
Comment (gruyaume) [reply README.md:106]: Some of it I simply moved over to `CONTRIBUTNG.md` but for the Security, Contributing and License subbullets, yeah I may have been heavy handed. I'll add them back

Comment (gruyaume) [reply README.md]: Done

**Review by natalian98 — DISMISSED** [review]
lgtm, integration tests are failing due to juju version

**Review by nsklikas — DISMISSED** [review]

**Comment by nsklikas** [comment]
Moved this to https://github.com/canonical/hydra-operator/pull/41

**Pull request closed without merging**